src/python/impactx/madx_to_impactx.py

```diff
@@ -192,6 +192,6 @@ def read_beam(ref: RefPart, madx_file):
 
     ref.set_charge_qe(ref_particle_dict["charge"])
     ref.set_mass_MeV(ref_particle_dict["mass"])
-    ref.set_energy_MeV(ref_particle_dict["energy"])
+    ref.set_kin_energy_MeV(ref_particle_dict["energy"] - ref_particle_dict["mass"])
```

Comment:
This is a bug-fix for low-energy particles in MAD-X files.
MAD-X specifies *total* particle energy in GeV.